docs/entity_light.md

```diff
@@ -3,14 +3,123 @@ title: Light Entity
 sidebar_label: Light
 ---
 
-> This entry is incomplete. Contribution welcome.
 
-## Properties
+A light entity is a device that controls the brightness, RGB value,color temperature and effects of a light source.
 
-> Properties should always only return information from memory and not do I/O (like network requests). Implement `update()` or `async_update()` to fetch data.
+## Properties
 
 | Name | Type | Default | Description
-| ---- | ---- | ------- | -----------
+| ---- | ---- | ---- | ----
+| brightness | int | None | Return the brightness of this light between 0..255
+| color_temp | int | None | Return the CT color value in mireds.
+| effect | String | None | Return the current effect.
+| effect_list | list | None | Return the list of supported effects.
+| hs_color | list | None | Return the hue and saturation color value [float, float].
+| max_minreds | int | int | Return the warmest color_temp that this light supports.
+| min_mireds | int | int | Return the coldest color_temp that this light supports.
+| state_attributes | list | list | Return optional state attributes.
```

Comment:
This is not a property that people are allowed to override

Reply:
The state attributes is the one that is not a property that people are to allowed override?